### ISSUE TITLE
Add 3DModelsCC0 for Graphics?

### ISSUE THREAD

**Issue by nisuaija** [libreprojects/libreprojects#203]
Library of public domain 3d models.

https://www.3dmodelscc0.com/

**Comment by ccoenen** [comment]
It's a great collection, but I would not file it under "software project". These are more like assets to be used in a software project, right?

**Comment by ironbishop** [comment]
yes but... we do list Wikipedia and Commons, and those are not software (MediaWiki is).
